server/app.js

```diff
@@ -264,6 +264,7 @@ const templateFn = require('./../client/template.js');
 app.use((req, res)=>{
 	const props = {
 		version     : require('./../package.json').version,
+		publicUrl   : config.get('publicUrl') ?? 'https://homebrewery.naturalcrit.com',

```

Comment:
```suggestion
		publicUrl   : config.get('publicUrl') ?? '',
```

Then move `https://homebrewery.naturalcrit.com` into the actual `default.json` config file. That way we can avoid having the URL hard-coded into the source.

Reply:
That should be implemented now.